x-pack/plugins/security_solution/server/endpoint/services/actions/clients/lib/base_response_actions_client.ts

```diff
@@ -154,6 +158,15 @@ export interface FetchActionResponseEsDocsResponse<
   [agentId: string]: LogsEndpointActionResponse<TOutputContent, TMeta>;
 }
 
+export interface ResponseActionsClientPendingAction<
```

Comment:
Changes the return value from `fetchAllPendingActions()` in preparation for supporting actions being sent to multiple agents.  the list of pending agent IDs are now returned enabling consumers of this data to have the pending list of agents already calculated